app/Controllers/configureController.php

```diff
@@ -74,14 +74,23 @@ public function displayAction(): void {
 			FreshRSS_Context::userConf()->bottomline_date = Minz_Request::paramBoolean('bottomline_date');
 			FreshRSS_Context::userConf()->bottomline_link = Minz_Request::paramBoolean('bottomline_link');
 			FreshRSS_Context::userConf()->show_nav_buttons = Minz_Request::paramBoolean('show_nav_buttons');
-			FreshRSS_Context::userConf()->html5_notif_timeout = Minz_Request::paramInt('html5_notif_timeout');
+			FreshRSS_Context::userConf()->html5_notif_timeout =
+				Minz_Request::paramInt('html5_notif_timeout') < 0 ? 0 : Minz_Request::paramInt('html5_notif_timeout');
```

Comment:
```suggestion
			FreshRSS_Context::userConf()->html5_notif_timeout = max(0, Minz_Request::paramInt('html5_notif_timeout'));
```

Maybe a less verbose syntax, also for the lines below

Reply:
nice trick. It is a bit tricky to read but very smart. Thanks!